File: app/src/main/java/br/com/santander/testeandroid/contact/model/CellResponse.java
package br.com.santander.testeandroid.contact.model;

import com.google.gson.annotations.SerializedName;

public class CellResponse {
    @SerializedName("id")
    private Integer id;

    @SerializedName("type")
    private Integer type;

    @SerializedName("message")
    private String message;

    @SerializedName("typefield")
    private String typeField;

    @SerializedName("hidden")
    private Boolean hidden;

    @SerializedName("topSpacing")
    private Double topSpacing;

    @SerializedName("show")
    private Integer show;

    @SerializedName("required")
    private Boolean required;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getType() {
        return type;
    }

    public void setType(Integer type) {
        this.type = type;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getTypeField() {
        return typeField;
    }

    public void setTypeField(String typeField) {
        this.typeField = typeField;
    }

    public Boolean getHidden() {
        return hidden;
    }

    public void setHidden(Boolean hidden) {
        this.hidden = hidden;
    }

    public Double getTopSpacing() {
        return topSpacing;
    }

    public void setTopSpacing(Double topSpacing) {
        this.topSpacing = topSpacing;
    }

    public Integer getShow() {
        return show;
    }

    public void setShow(Integer show) {
        this.show = show;
    }

    public Boolean getRequired() {
        return required;
    }

    public void setRequired(Boolean required) {
        this.required = required;
    }
}

File: app/src/main/java/com/vctapps/santanderchallenge/form/data/entity/CellResponse.kt
package com.vctapps.santanderchallenge.form.data.entity

import com.google.gson.annotations.SerializedName

data class CellResponse(@SerializedName("id")         val id: Int?,
                        @SerializedName("type")       val type: Int?,
                        @SerializedName("message")    val message: String?,
                        @SerializedName("typefield")  val typeField: Any?,
                        @SerializedName("hidden")     val hidden: Boolean?,
                        @SerializedName("topSpacing") val topSpacing: Double?,
                        @SerializedName("show")       val show: Int?,
                        @SerializedName("required")   val required: Boolean?)